tests/test_two_view_estimator.py

```diff
@@ -0,0 +1,71 @@
+"""Unit tests for the two-view estimator.
+
+Authors: Ayush Baid
+"""
+from gtsfm.common.keypoints import Keypoints
+import unittest
+
+import gtsam
+import numpy as np
+from gtsam import Cal3Bundler, EssentialMatrix, Pose3, Unit3
+
+import gtsfm.utils.geometry_comparisons as comp_utils
+import gtsfm.utils.io as io_utils
+from gtsfm.two_view_estimator import TwoViewEstimator
+
+GTSAM_EXAMPLE_FILE = "18pointExample1.txt"
+EXAMPLE_DATA = io_utils.read_bal(gtsam.findExampleDataFile(GTSAM_EXAMPLE_FILE))
+
+
+class TestTwoViewEstimator(unittest.TestCase):
+    """Unit tests for the 2-view estimator"""
```

Comment:
thanks for adding the test. could you add a sentence about the data you're using for the test, and why you chose it?

Reply:
Comments for each code block would also be useful.